src/data/technology/index.ts

```diff
@@ -0,0 +1,20 @@
+import { importAll, expandRelation } from '../helpers'
+import * as R from 'ramda'
+import { importAllNode } from '../helpers-node'
+import { Technology } from './types'
+
+const loadData = (): Technology[] => {
+  try {
+    const context = require.context('./', false, /\.json$/)
+    return importAll<Technology>(context)
+  } catch (error) {
+    return importAllNode<Technology>(__dirname)
+  }
+}
+
+const data = loadData()
+
+export const expandTechnologies = R.partial(expandRelation, [
```

Comment:
This belongs in the home data/projects/index.tsx file, not here. since the path ('usedTechnologies') describes the project object and it's also called there